libs/core/testing/include/hpx/modules/testing.hpp

```diff
@@ -174,6 +179,64 @@ namespace hpx { namespace util {
 
         HPX_CORE_EXPORT extern fixture global_fixture;
 
+        // Json output for performance reports
```

Comment:
I'd perhaps move this functionality to a separate header, even if it's just to include it in this file.